ts/features/fci/components/SuccessComponent.tsx

```diff
@@ -20,30 +24,51 @@ const SuccessComponent = (props: {
 }) => {
   const now = new Date();
   const expires_at = new Date(props.signatureRequest.expires_at);
+  const updated_at = new Date(props.signatureRequest.updated_at);
   const status = props.signatureRequest.status;
   const dispatch = useIODispatch();
 
-  // the signature request is expired
-  if (expires_at < now) {
+  // if the user (signer) has not signed and the request is expired
+  // the user can no longer sign anymore
+  if (
+    (status === SignatureRequestDetailStatus.WAIT_FOR_SIGNATURE ||
+      status === SignatureRequestDetailStatus.REJECTED) &&
+    expires_at < now
+  ) {
     return (
       <ErrorComponent
-        title={I18n.t("features.fci.errors.expired.title")}
-        subTitle={I18n.t("features.fci.errors.expired.subTitle")}
+        title={I18n.t("features.fci.errors.expiredAfterSigned.title")}
+        subTitle={I18n.t("features.fci.errors.expiredAfterSigned.subTitle")}
```

Comment:
Here the status is not signed yet so I guess the error message must be:
```suggestion
        title={I18n.t("features.fci.errors.expired.title")}
        subTitle={I18n.t("features.fci.errors.expired.subTitle")}
```